enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,987 @@
+
+
+#### Unmanaged Subnet Annotation Migration Workflow
+
+Migrating an unmanaged `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+service annotation to `spec.endpointPublishingStrategy.loadBalancer.providerParameters.aws.networkLoadBalancerParameters.eipAllocations`
+after upgrading to 4.y doesn't require a cluster admin to delete the LoadBalancer-type
+service:
+
+1. Cluster admin confirms that initially `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`
+   is set on the LoadBalancer-type service managed by the IngressController.
+2. The cluster admin upgrades OpenShift to v4.y and the service annotation is not
+   changed or removed. For more details see [section](#Any change to the service annotation does not trigger the operator to manage the load balancer service).  
+3. After upgrading, the IngressController will emit a `LoadBalancerProgressing` condition
+   with `Status: True` because the spec's `EIPAllocations` (an empty slice `[]`) does not equal
+   the current annotation.
+4. In this case, the cluster admin must directly update
+   `spec.endpointPublishingStrategy.loadBalancer.providerParameters.aws.networkLoadBalancerParameters.eipAllocations` to the
+   current value of the `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` service.
+5. The Ingress Operator will resolve the `LoadBalancerProgressing` condition back to
+   `Status: False` as long as `EIPAllocations` is equivalent to `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+Currently, there is no unique consideration for making this change work with Hypershift.
+
+#### Standalone Clusters
+
+This proposal does not have any special implications for standalone clusters.
+
+#### Single-node Deployments or MicroShift
+
+This proposal does not have any special implications for single-node
+deployments or MicroShift.
+
+### Implementation Details/Notes/Constraints
+
+#### 1. Set EIP through installer for the default ingress controller:
+1. The admin sets the `eipAllocations` in the installer using the install-config.yaml.
+
+2. The installer then sets the ingress cluster object with the `eipAllocations`. 
+
+3. The cluster ingress operator then creates a default ingress controller by:
+     
+     1. creating an default ingress controller CR.
+
+     2. then scaling a deployment for the Ingress Controller CR which has the value of the `eipAllocations` from the 
+     ingress cluster object.
+
+     3. then creating a service of service type load balancer with the annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations`, 
+     which uses the value from the field `eipAllocations` of ingress controller CR.
+
+#### 2. Create a new ingress controller with EIPs:
+   
+   The admin sets the `eipAllocations` in the Ingress Controller CR. Then same process
+   as per point 1.c.ii and 1.c.iii is followed by the ingress operator. 
```

Comment:
Based on your changes above, this needs to be:

```suggestion
   as per point 1.3.ii and 1.3.iii is followed by the ingress operator. 
```

Reply:
I think it should be "as per point 1.iii.b and 1.iii.c is followed by the ingress operator. "